src/main/java/bio/terra/flight/study/create/CreateStudyPrimaryDataStep.java

```diff
@@ -19,13 +21,16 @@ public CreateStudyPrimaryDataStep(BigQueryPdao bigQueryPdao) {
 
     Study getStudy(FlightContext context) {
         FlightMap inputParameters = context.getInputParameters();
-        StudyRequestModel studyRequest = inputParameters.get("request", StudyRequestModel.class);
+        StudyRequestModel studyRequest = inputParameters.get(JobMapKeys.REQUEST.getKeyName(), StudyRequestModel.class);
         return StudyJsonConversion.studyRequestToStudy(studyRequest);
     }
 
     @Override
     public StepResult doStep(FlightContext context) {
-        bigQueryPdao.createStudy(getStudy(context));
+        Study study = getStudy(context);
+        bigQueryPdao.createStudy(study);
```

Comment:
did we need to separate those 2 lines? not that it really matters - i was just expecting you to need the study for something else below

Reply:
Sorry to not address this yesterday.. I think I did this while I was debugging. It's not necessary, but I think it's okay to leave in since it improves readability.